ycms/cms/templates/patient/patient_card.html

```diff
@@ -63,7 +63,8 @@ <h3 class="text-lg font-semibold dark:text-white mb-2">{% translate "Hostpital S
             {% if not assignment_mode %}
                 <div class="flex justify-end border-t pt-2 border-gray-200 rounded-b dark:border-gray-600">
                     <div class="flex gap-2">
-                        <a href="{% url 'cms:protected:assign_patient' ward_id=ward.id assignment_id=bed_assignment.id %}">
+                        {% url 'cms:protected:assign_patient' ward_id=ward.id assignment_id=bed_assignment.id as url %}
+                        <a href="{{ url }}">
```

Comment:
Same here - do correct me if I'm wrong though!!
```suggestion
                        <a href="{% url 'cms:protected:assign_patient' ward_id=ward.id assignment_id=bed_assignment.id %}">
```